src/settings/creator/Containers/Info.svelte

```diff
@@ -177,22 +137,22 @@
                 <div use:folder />
             </TextComponent>
             <ToggleComponent
-                name={"Support Timelines Events"}
-                desc={timelinesDesc}
-                value={timelines}
+                name={"Support Inline Events"}
+                desc={"Look for <span> tags defining events in notes."}
+                value={inlineEvents}
                 on:click={() => {
-                    $calendar.supportTimelines = !$calendar.supportTimelines;
+                    $calendar.supportInlineEvents = !$calendar.supportInlineEvents;
                 }}
             />
-            {#if timelines}
-                {#key $calendar.syncTimelines}
+            {#if inlineEvents}
+                {#key $calendar.inlineEvents}
```

Comment:
This should probably be `supportInlineEvents`

Reply:
This isn't resolved; `inlineEvents` does not exist on the `Calendar` interface.

Reply:
In Svelte, a store's underlying data can be accessed using the `$` prefix.

Essentially, `$` does something like this:

```js
let store = writable(calendar);

//$store resolves to:
let $store;
store.subscribe((val) => $store = val);
```

So `$calendar` is the `Calendar` object that we created a store from.